.github/workflows/pull_request_checks.yml

```diff
@@ -66,7 +66,9 @@ jobs:
 
       # Get all previous enum definitions for changed models from master
       - name: Get previous enum definitions
-        run: bundle exec ruby /tmp/enums_in_models.rb enums_after.log >enums_before.log
+        run: |
+          bundle install
```

Comment:
If we have gem updates and checkout master before this step, we aren't finding the gems required. I've added another bundle step to pull in the gems again, however will investigate if we can utilise the bundle cache somehow and invalidate if it is different between branches so we don't pull those in everytime

Reply:
Just double checked: https://github.com/ruby/setup-ruby#caching-bundle-install-manually and it seems its best to just rerun this as it doesn't add more than ~ 10s to the build